src/content/blog/2025-plans.md

```diff
@@ -50,7 +50,7 @@ We'd love to have your input as daily users as we work through these this month.
 6. [Improve self-diagnose capability](https://github.com/ddev/ddev/issues/6461) . We currently have ddev debug test but it would be great to implement something that was readable and actionable for ordinary mortals.
 7. [DDEV's Message-of-the-day and ddev.com should show current funding status and need](https://github.com/ddev/ddev/issues/6892). We now have an automatically updated JSON feed that can make this possible.
 8. DDEV Windows/WSL2 packaging and installation: The traditional Windows installer needs work, and the WSL2 install scripts are written in hard-to-maintain PowerShell. This work can be consolidated and improved, including [improving the Windows hosts-file escalation technique](https://github.com/ddev/ddev/issues/6440).
-9. [Rewrite `ddev launch` in Golang instead of as script](https://github.com/ddev/ddev/issues/6394).
+9. [Change `ddev launch` to a more configurable script-based option](https://github.com/ddev/ddev/issues/6441). Instead of always using `ngrok` there should be multiple ways to share.
```

Comment:
Oh, it should be `ddev share` not `ddev launch`

Reply:
Thanks :) Broken brain. Will get it fixed.